packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/Evm-artifacts/L1Bridge.bin

```diff
@@ -0,0 +1 @@
0000000000000000000081525060200191505060405180910390fd5b6000612d30612c7d565b905060018160000160008473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff0219169083151502179055505050565b600080823b905060008111915050919050565b612dc56040518060600160405280602981526020016132466029913982612ff8565b50565b612dea6040518060600160405280602881526020016131626028913982612ff8565b50565b600080826040516020018082805190602001908083835b60208310612e275780518252602082019150602081019050602083039250612e04565b6001836020036101000a03801982511681845116808217855250505050505090500191505060405160208183030381529060405280519060200120905080915050919050565b600080826040516020018082805190602001908083835b60208310612ea75780518252602082019150602081019050602083039250612e84565b6001836020036101000a0380198251168184511680821785525050505050509050019150506040516020818303038152906040528051906020012090508054915050919050565b6000612ef983611884565b14612f6c576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252600b8152602001807f414c52454144595f53455400000000000000000000000000000000000000000081525060200191505060405180910390fd5b612f768282612f7a565b5050565b6000826040516020018082805190602001908083835b60208310612fb35780518252602082019150602081019050602083039250612f90565b6001836020036101000a038019825116818451168082178552505050505050905001915050604051602081830303815290604052805190602001209050818155505050565b600073ffffffffffffffffffffffffffffffffffffffff1661301983612e6d565b73ffffffffffffffffffffffffffffffffffffffff16146130a2576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252600b8152602001807f414c52454144595f53455400000000000000000000000000000000000000000081525060200191505060405180910390fd5b6130ac82826130b0565b5050565b6000826040516020018082805190602001908083835b602083106130e957805182526020820191506020810190506020830392506130c6565b6001836020036101000a038019825116818451168082178552505050505050905001915050604051602081830303815290604052805190602001209050818155505050565b6000808585111561313e57600080fd5b8386111561314b57600080fd5b600185028301915084860390509450949250505056fe535441524b4e45545f544f4b454e5f4252494447455f4d4553534147494e475f434f4e54524143545452414e534645525f544f5f535441524b4e45545f414d4f554e545f4558434545444544535441524b4e45545f544f4b454e5f4252494447455f4c325f544f4b454e5f434f4e5452414354535441524b4e45545f544f4b454e5f4252494447455f4d41585f544f54414c5f42414c414e4345535441524b4e45545f544f4b454e5f4252494447455f4445504f5349544f525f414444524553534553535441524b4e45545f544f4b454e5f4252494447455f4d41585f4445504f534954535441524b4e45545f45524332305f544f4b454e5f4252494447455f544f4b454e5f41444452455353537461726b576172655f537461726b6e65744574684272696467655f323032325f31a264697066735822122093438020cfb8664038b3d76ef7753def54b279ffcd3aea74e05d88704a3cd63764736f6c634300060c0033535441524b574152455f44454641554c545f474f5645524e414e43455f494e464f
```

Comment:
Why did you include this .bin file here? Doesn't seem used, let's rebase it out of this PR and our git history.